README-TEST.md

```diff
@@ -58,6 +58,11 @@ Run end to end test on Apache 2.2.0 without stress test, with ssl
 	./run_test_apache.sh 2.2.0 ./apache_properties_ssl false true
 ```
 
+There is also option to run selected tests locally using additional named parameters:
+```
+./run_test_<confluent/apache>.sh <version> <path to apache config folder> <stress> <ssl> --local=true --tests=TestStringJson,TestAvroAvro
```

Comment:
Can you explain in the README what `--local` is responsible for?

Reply:
Added information about optional flags